TicketBookingAPI/Middlewares/ExceptionMiddlewareExtension.cs

```diff
@@ -0,0 +1,58 @@
+﻿using AutoWrapper.Wrappers;
+using Microsoft.AspNetCore.Diagnostics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using TicketBooking.Common.AppExceptions;
+using TicketBookingAPI.Controllers;
+
+namespace TicketBooking.Common.Middlewares
+{
+    public class ExceptionMiddlewareExtension
+    {
+        private readonly RequestDelegate _next;
+        private readonly ILogger _logger;
+
+        public ExceptionMiddlewareExtension(RequestDelegate next, ILogger logger)
```

Comment:
rename to HandleExceptionMiddleware.
The purpose of this middleware is logging the error detail for easy to investigate the error. Otherwise can custom the response for client incase error.